addon/helpers/replace-with-external.js

```diff
@@ -43,10 +43,8 @@ export default Helper.extend({
 			_params[_params.length - 1] = { queryParams: queryParams.values };
 		}
 
-		return function(value) {
-			router.replaceWith(..._params);
-
-			return value;
```

Comment:
What purpose did this return value have and is removing it problematic? I couldn't see any issue with doing so but maybe there's a reason it was here 😅

Reply:
Thanks @adrigzr! Yes, they seem to have moved the routing packing internally in `ember-source`... this would've broken anyway 😬

Reply:
Yep, it was it.

Could you try to merge with this branch? #12

Reply:
Ah great, thanks @adrigzr! 🙌